test/services/tokens.test.js

```diff
@@ -329,15 +328,14 @@ describe('Access Token tests', function () {
           should.exist(tokens);
           tokens.length.should.eql(tokenObjs.length * 2);
           tokens.forEach(tokenObj => {
-            tokenObj.prop.should.be.oneOf(_.map(tokenObjs, 'prop'));
+            tokenObj.prop.should.be.oneOf(tokenObjs.map(x => x.prop));
           });
-          _.map(tokens, 'archived').filter(val => val).length.should.eql(4);
+          tokens
+          .map(x => x.archived)
+          .filter(val => val).length.should.eql(4);
```

Comment:
What's the aim behind `filter(val => val)`?

Reply:
haha, no idea, just copy paste 👍

Reply:
actually it is required, it is checking only archived tokens, and ensures that the tokens are indeed archived.